nf_core/list.py

```diff
@@ -60,19 +60,15 @@ def get_remote_workflows(self):
 
         # List all repositories at nf-core
         logging.debug("Fetching list of nf-core workflows")
-        gh_api_url = 'https://api.github.com/orgs/nf-core/repos?per_page=100'
-        response = requests.get(gh_api_url, timeout=10)
+        #gh_api_url = 'https://api.github.com/orgs/nf-core/repos?per_page=100'
```

Comment:
Could you remove this please?

Reply:
ah thanks, i have overseen this ! will change that